src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employeesContainer = document.querySelector('ul');
+const employees = [...document.querySelectorAll('[data-salary]')];
+const employeesList = employees.map((employee) => {
+  return {
+    name: employee.innerText,
+    position: employee.dataset.position,
+    salary: +employee.dataset.salary.replace(/[$,]/g, ''),
+    age: employee.dataset.age,
```

Comment:
The 'age' property is being stored as a string since it's directly taken from the data attribute without conversion. If you need to perform numerical operations on age, you should convert it to a number.